vnext/Microsoft.ReactNative.IntegrationTests/ReactNativeHostTests.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+
+#include <winrt/Microsoft.ReactNative.h>
+
+namespace ReactNativeIntegrationTests {
+
+TEST_CLASS (ReactNativeHostTests) {
```

Comment:
I noticed this with some other tests earlier, but have been wondering why we have been using Motif APIs for new code instead of raw GTest?

Reply:
Motif tests give us more flexibility to run in different environments. It is essentially a 'front-end' while the backend can be changed if needed. E.g. we can run the Motif tests on top of GTest, VS tests, Office Motif tests, and in future on top of XCTest.

---
In reply to: [413890737](https://github.com/microsoft/react-native-windows/pull/4690#discussion_r413890737) [](ancestors = 413890737)

Reply:
My concern is more that Motif isn't particularly well documented externally, or particularly feature rich. We want to encourage those outside of Office familiarity to add to tests, right?

Reply:
dumb question...what is motif? I couldn't find anything on it doing a quick search

Reply:
@asklar Microsoft Office Test Integration Framework, later rebranded as "MS Open Test Framework"